imports/plugins/core/orders/client/components/orderSummary.js

```diff
@@ -149,15 +156,24 @@ class OrderSummary extends Component {
         <div className="order-summary-form-group">
           <strong data-i18n="orderShipping.shipTo">Ship to</strong>
           <div className="invoice-details">
-            <strong>Phone: </strong>{profileShippingAddress.phone}
+            <strong>Phone: </strong>{profileShippingAddress && profileShippingAddress.phone}
           </div>
         </div>
 
         <div style={{ marginTop: 4 }}>
-          <span>{profileShippingAddress.fullName}</span>
-          <br/><span>{profileShippingAddress.address1}</span>
-          {profileShippingAddress.address2 && <span><br/>{profileShippingAddress.address2}</span>}
-          <br/><span>{profileShippingAddress.city}, {profileShippingAddress.region}, {profileShippingAddress.country} {profileShippingAddress.postal}</span>
+          <span>{profileShippingAddress && profileShippingAddress.fullName}</span>
+          <br/>
+          <span>{profileShippingAddress && profileShippingAddress.address1}</span>
+          {profileShippingAddress && profileShippingAddress.address2 &&
+            <span><br/>{profileShippingAddress.address2}</span>
+          }
+          <br/>
+          <span>
+            {profileShippingAddress && profileShippingAddress.city},&nbsp;
```

Comment:
This works. 👍 

You _could_ have setup `profileShippingAddress` to pass an empty object if undefined and then just used `profileShippingAddress.property` everywhere, but that's not necessarily better, just different. Not blocking.

Reply:
Yeah, this makes sense. Avoids having to repeat `profileShippingAddress &&` in a lot of places.